clang/include/clang/Driver/Options.td

```diff
@@ -3394,6 +3394,10 @@ def fno_inline_functions : Flag<["-"], "fno-inline-functions">, Group<f_clang_Gr
   Visibility<[ClangOption, CC1Option]>;
 def fno_inline : Flag<["-"], "fno-inline">, Group<f_clang_Group>,
   Visibility<[ClangOption, CC1Option]>;
+def fpartial_inlining : Flag<["-"], "fpartial-inlining">, Group<f_clang_Group>,
```

Comment:
These 100% need `HelpText` sections.  Also, this should use the `BoolFOption` infrastructure instead I think.

Reply:
The `BoolFOption` infrastructure maps frontend arguments onto `CompilerInvocation`. The intent of this PR seems to be to add driver arguments that map to the `-mllvm` frontend argument (which already does map to `CompilerInvocation`). I think these should not be `CC1Option`, otherwise LGTM.